### PR TITLE
Feature/3718 task submission flow add alerts

### PR DESCRIPTION
**Pull request by KaiTeuber**
## Description
If the user try to cancel the submission flow new alerts will get shown

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-3718

## Screenshot
Screen Shot 2020-11-27 at 17 36 51
Screen Shot 2020-11-27 at 17 37 18

### CODE REVIEW

File: src/xcode/ENA/ENA/Resources/Storyboards/ExposureSubmission.storyboard [hunk 1]
```diff
@@ -213,26 +213,6 @@
             </objects>
             <point key="canvasLocation" x="3" y="1185"/>
         </scene>
-        <!--Exposure Submission Symptoms View Controller-->
-        <scene sceneID="Lgw-rw-O6o">
```
Comment (marcussc) [inline src/xcode/ENA/ENA/Resources/Storyboards/ExposureSubmission.storyboard:217]: <3 <3 <3 <3

**Review by whiskey — APPROVED** [review]

File: src/xcode/ENA/ENA/Resources/Storyboards/ExposureSubmission.storyboard [hunk 2]
```diff
@@ -213,26 +213,6 @@
...
-                        <userDefinedRuntimeAttribute type="color" keyPath="cellBackgroundColor">
-                            <color key="value" white="0.0" alpha="0.0" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
-                        </userDefinedRuntimeAttribute>
-                    </userDefinedRuntimeAttributes>
-                </viewController>
-                <placeholder placeholderIdentifier="IBFirstResponder" id="R6e-pz-aeF" userLabel="First Responder" customClass="UIResponder" sceneMemberID="firstResponder"/>
-            </objects>
-            <point key="canvasLocation" x="6525" y="155"/>
-        </scene>
         <!--Exposure Submission Symptoms Onset View Controller-->
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Resources/Storyboards/ExposureSubmission.storyboard:216]: We should also be able to delete this one, right? 😁

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift [hunk 3]
```diff
@@ -418,9 +420,34 @@ extension ExposureSubmissionCoordinator {
 				)
 			}
 		)
-
 		push(vc)
 	}
+	
+	func presentSubmissionSymptomsCancelAlert() {
+		guard let navigationController = navigationController else {
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift:427]: Could we just use `navigationController?.present`?

**Review by nickguendling — CHANGES_REQUESTED** [review]

**Review by nickguendling — APPROVED** [review]

**Review by janetback — APPROVED** [review]
UA approved

**Pull request merged**